x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A common set of SGML entities and XML/CSS style sheets"
+DESCRIPTION="This package provides a common set of SGML entities and XML/CSS \
+style sheets used in building/formatting the documentation provided in other\
+X.Org packages.   It's typically only needed by people building from \
+source who want to produce formatted documentation from their builds, \
+or those who have installed the HTML version of the documentation, \
+which refers to the included common xorg.css stylesheet."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
+COPYRIGHT="2010, Oracle and/or its affiliates
+	2005 Red Hat, Inc."
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/freedesktop/xorg-sgml-doctools/archive/xorg-sgml-doctools-$portVersion.tar.gz"
```

Comment:
I think it's better to fetch the tarball from it's home :)

https://www.x.org/releases/individual/doc/xorg-sgml-doctools-1.11.tar.bz2